docs/Antora/modules/howto_guides/pages/howto_configure_instance.adoc

```diff
@@ -0,0 +1,291 @@
+= Instance configuration
+
+The Instance is the top-level openDAQ(TM) object, serving as a container for the openDAQ context and the base module manager.
+
+**Related articles:**
+
+  * xref:background_info:opendaq_architecture.adoc#instance[Architecture - Instance]
+
+== Creating the default openDAQ(TM) Instance
+Upon creation, it generates a Client device, which is a default device implementation capable of loading any function blocks present in the module manager's search path. If the native openDAQ client module is loaded, the Client device can connect to any TMS-enabled device using the `addDevice` function. The Client is set as the root device when the instance is created.
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+int main(int argc, const char* argv[])
+{
+    daq::InstancePtr instance = daq::Instance();
+    daq::DevicePtr device = instance.addDevice("daq.opcua://127.0.0.1/");
+
+    return 0;
+}
+----
+====
+
+== Creating openDAQ(TM) Instance with Instance Builder
+
+openDAQ(TM) provides an advanced way to configure an Instance using the Instance Builder. With the Instance Builder, developers can configure the Logger, Scheduler, Module Manager, and Root Device by setting custom modules or modifying default objects.
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+int main(int argc, const char* argv[])
+{
+    daq::InstanceBuilderPtr instanceBuilder = daq::InstanceBuilder()
+    daq::InstancePtr instance = instanceBuilder.build();
+    daq::DevicePtr device = instance.addDevice("daq.opcua://127.0.0.1/");
```

Comment:
Again, I'd leave out the `addDevice` and other instance calls here. I'd explain what happens when you call `build`. In this case I'd mention that the defaults are used, and what the defaults are.